README.md

```diff
@@ -1,39 +1,17 @@
 # cf-remote
 
-## Installation
-
-cf-remote requires python 3 with fabric and requests.
-(Install using brew/apt/yum/pip).
-It's a _normal_ python package, but doesn't have a setup script (yet).
-
-Clone `cfengine/core` repo where you want, and then symlink, like this:
-
-```
-$ git clone https://github.com/cfengine/cf-remote.git
-$ ln -s `pwd`/cf-remote/cf_remote/__main__.py /usr/local/bin/cf-remote
-```
+## Requirements
 
-Install dependencies:
+cf-remote requires python 3.6 or greater.
 
-```
-$ cd cf-remote/
-$ pip3 install -r requirements.txt
-```
+## Installation
 
-Check that it worked:
+Install with pip3:
 
 ```
-$ ls -al `which cf-remote`
-lrwxr-xr-x  1 olehermanse  admin  68 Jan 10 15:20 /usr/local/bin/cf-remote -> /northern.tech/cfengine/cf-remote/cf_remote/__main__.py
-$ cf-remote --version
-cf-remote version 0.1 (BETA)
-Available CFEngine versions:
-3.13.0, 3.12.1, 3.12.0, 3.10.5, 3.10.4, 3.10.3, 3.10.2, 3.10.1, 3.10.0
+$ pip3 install cf-remote
```

Comment:
This does not work yet, right?

Reply:
Correct. But if we release with this change then it will work. If the release github action doesn't work we can always manually upload it.

Reply:
The README.md content will go into the pypi entry so we probably want these changes in the build we push up.

Reply:
Not a huge deal, but I would prefer to see it working, and then publish another version where instructions are updated.

Reply:
I'm okay with having the instructions if the release is the only missing bit and it will just work after the release.

Reply:
> I'm okay with having the instructions if the release is the only missing bit and it will just work after the release.

Well, this is what we don't really know yet, since we never did it ;)